src/features/Main/sections/Hero/index.tsx

```diff
@@ -1,6 +1,38 @@
+import Image from 'next/image';
+import Link from 'next/link';
+import HeroImage from '../../../../../public/assets/img/Hero.svg';
 import { FC } from 'react';
+import { SocialMedia } from '@/constants/socialMedia';
+
 
 const Hero: FC = () => {
-  return <div>Hero</div>;
+  return (
+    <div className="container mx-auto px-4 flex flex-col md:flex-row items-center justify-between my-6">
+      <div className=" md:w-2/5 flex flex-col gap-6">
+        <h1 className="text-5xl font-semibold text-[#0F3665]">
+          Improve Your Skill by Talking High-Quality Classes
+        </h1>
+        <p className="text-sm">
+          <strong>HMCLearn</strong> is a streamlined and user-friendly Learning
```

Comment:
mungkin prefer pakai tag `<b>` :)

reference: https://www.seobility.net/en/wiki/Strong_and_Bold_Tags#:~:text=The%20difference%20between%20the%20two,word%20or%20section%20of%20text.

Reply:
okey kak